charts/orchestrator/values.yaml

```diff
@@ -79,3 +84,5 @@ tekton:
 argocd:
   enabled: false # whether to install the ArgoCD plugin and create the orchestrator AppProject
   namespace: "" # Defines the namespace where the orchestrator's instance of ArgoCD is deployed. The value is captured when running setup.sh script and stored as a label in the selected namespace. User can override the value by populating this field. Defaults to `orchestrator-gitops` in the setup.sh script.
+
+isReleaseCandidate: false # Indicates RC builds should be used by the chart to install RHDH and Sonataflow
```

Comment:
I think this field belongs to the RHDH and SonataFlow sections. It's not a global value so it's scope should be defined where it applies.

Reply:
moved under each section (rhdh/sonataflow).